src/audio/base_fw_intel.c

```diff
@@ -308,6 +387,8 @@ int basefw_vendor_set_large_config(struct comp_dev *dev,
 	switch (param_id) {
 	case IPC4_FW_CONFIG:
 		return basefw_set_fw_config(first_block, last_block, data_offset, data);
+	case IPC4_DMA_CONTROL:
```

Comment:
@tmleman FYI: looks like we have 2 PR's with IPC4_DMA_CONTROL
https://github.com/thesofproject/sof/pull/9156

Reply:
Aligned this PR to [basefw: Add handling of IPC4_DMA_CONTROL messages](https://github.com/thesofproject/sof/pull/9156)